https://issues.redhat.com/browse/ACM-17809 (#7508)
open-cluster-management/rhacm-docs: release_notes/acm_whats_new.adoc

[#dr4hub-whats-new]
== Backup and restore

> * You can now install VolSync as a deployment on the `volsync-system` namespace on the managed cluster. In earlier versions, you needed to install VolSync as an {olm} operator. By installing VolSync as a deployment on the `volsync-system` namespace, you have support for non-OpenShift managed clusters. 

Reviewer: I think again I would prefer to reword a bit.

I'm trying to get across that the main changes in ACM 2.13 and beyond with volsync are:
- You can now deploy volsync from ACM onto ACM managed clusters even when those managed clusters are not openshift.
- Previously VolSync was deployed as an OLM operator on openshift clusters.  Now it will instead appear as a deployment in the `volsync-system` namespace. 

I'd prefer not to say you "can install it as a deployment", as the user is not actually creating a deployment and don't want to be confusing - they are deploying VolSync from ACM as a managed cluster addon (as has always been done - so we don't need to spell this part out in the whats new) - but the end result is no longer an OLM operator with CSV that they see on the managed cluster.  The Volsync operator is still installed, it' s just running in the `volsync-system` namespace now.

> * In earlier versions, you had to install VolSync as an {olm} operator on {ocp-short} clusters. Now, when you install VolSync from {acm-short} on the managed cluster, it gets installed into the `volsync-system` namespace on that managed cluster. Now you can deploy VolSync from {acm-short} onto managed clusters, even if those clusters are not {ocp-short} clusters. 

Reviewer 2: ```suggestion
* When you install VolSync from {acm-short} on your managed cluster, VolSync is now installed in the `volsync-system` namespace on your managed cluster.

* You can now deploy VolSync from {acm-short} onto non-{ocp-short} managed clusters.
```

Should this be 2 separate points? I would also recommend linking to the VolSync docs for at least one of these bullet points.
